client-src/elements/chromedash-guide-stage-page.js

```diff
@@ -125,13 +126,18 @@ export class ChromedashGuideStagePage extends LitElement {
     setupScrollToHash(this);
   }
 
-  handleFormSubmit(event, hiddenTokenField) {
-    event.preventDefault();
+  handleFormSubmit(e, hiddenTokenField) {
+    e.preventDefault();
+    const submitBody = formatFeatureChanges(this.fieldValues, this.featureId);
 
     // get the XSRF token and update it if it's expired before submission
     window.csClient.ensureTokenIsValid().then(() => {
       hiddenTokenField.value = window.csClient.token;
-      event.target.submit();
+      return csClient.updateFeature(submitBody);
```

Comment:
Maybe in a follow-up CL, the 2 lines above this might be possible to eliminate.  IIRC, the hiddenTokenField was only needed because we were doing HTML form submission.  If we make an API call instead, csClient should handle the token for that without needing any token-management code here.